### PR TITLE
Add the Enterprise Sales Representative video to the candidate page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done

Add the Enterprise Sales Representative video to the candidate page
Could not use the hiring JSON as this is on the hiring lead level not the individual role
Updated the title text

## QA

- Check the code looks right as the position is not open yet.
- Check some candidate pages to check they continue to work as expected

### CODE REVIEW

**Comment by akbarkz** [comment]
@anthonydillon Is there an open position currently that we could use to test this change? I couldn't find one.

**Comment by anthonydillon** [comment]
Nope, as mentioned in the QA steps this position is not open yet. I'll try and assign a test candidate to the position but not sure I can if it's not open

File: webapp/application.py [hunk 1]
```diff
@@ -250,7 +250,11 @@ def _get_application(application_id):
             except HTTPError as error:
                 print(error)
 
-            if (
+            if job_id == "2680006":  # Enterprise Sales Representative
```
Comment (akbarkz) [inline webapp/application.py:253]: What's the chance of this ID changing in the future? Are we sure it will always have this ID?

**Review by akbarkz — APPROVED** [review]

**Comment by carkod** [comment]
@anthonydillon can you squash your "Fix lint" commits? They are not very useful in Git history.

File: webapp/application.py (hunk 1 above)
Comment (anthonydillon) [reply webapp/application.py:253]: It's the id of the role we want to display. There is no system of role-based videos only HL at this point so its hacky but might just be a one off.

**Pull request merged**